frontend/src/App.tsx

```diff
@@ -2,12 +2,14 @@ import { BrowserRouter, Routes, Route } from 'react-router-dom';
 import HomePage from '@/pages/home-page';
 import AddBlog from '@/pages/add-blog';
 import DetailsPage from '@/pages/details-page';
+import ScrollToTop from "@/components/skeletons/ScrollTop";
 function App() {
   return (
     <BrowserRouter>
+    <ScrollToTop />
       <Routes>
         <Route path="/">
-          <Route index element={<HomePage />} />
+          <Route index element={<HomePage />}/>
```

Comment:
Can you kindly run `npm run format` in the frontend directory, because it helps in readability